tensorflow_lite_support/python/task/audio/core/pybinds/audio_utils.cc

```diff
@@ -0,0 +1,71 @@
+/* Copyright 2022 The TensorFlow Authors. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+==============================================================================*/
+#include "tensorflow_lite_support/examples/task/audio/desktop/utils/audio_utils.h"
+
+#include "pybind11/pybind11.h"
+#include "pybind11_abseil/status_casters.h"  // from @pybind11_abseil
+
+namespace tflite {
+namespace task {
+namespace audio {
+
+namespace {
+namespace py = ::pybind11;
+
+}  //  namespace
+
+
+PYBIND11_MODULE(audio_utils, m) {
```

Comment:
@kinaryml  As we should get rid of the AudioData class, please move the logic to decode a WAV file to AudioBuffer into the audio_buffer pybinds.